Sources/XCTest/XCTestCase.swift

```diff
@@ -189,7 +189,7 @@ extension XCTestCase {
     ///   between these environments. To ensure compatibility of tests between
     ///   swift-corelibs-xctest and Apple XCTest, it is not recommended to pass
     ///   explicit values for `file` and `line`.
-    public func expectationWithDescription(description: String, file: StaticString = #file, line: UInt = #line) -> XCTestExpectation {
+    public func expectation(withDescription description: String, file: StaticString = #file, line: UInt = #line) -> XCTestExpectation {
```

Comment:
"With" is usually a vacuous preposition that does not carry any semantics.  I think this is one of those cases.  (Other cases in XCTest may also be like that.)